lms/djangoapps/teams/static/teams/templates/team-profile.underscore

```diff
@@ -48,7 +48,7 @@
 
         <% if (isMember) { %>
             <div class="leave-team">
-                <button class="btn btn-link btn-base btn-secondary leave-team-link"><%- gettext("Leave Team") %></button>
+                <button class="btn btn-link leave-team-link"><%- gettext("Leave Team") %></button>
```

Comment:
Both font-weight and font-size rules are being applied from the base <button> element styling here.  I can take a look and propose an update to resolve this.